ui/app/components/ui/popover/index.scss

```diff
@@ -61,7 +61,7 @@
     &__button {
       background: none;
       font-size: inherit;
-      padding: 0;
+      padding: 0 0 0 10px;
```

Comment:
Ah, I anticipated this question, but failed to submit the comment.

This is to avoid the Popover title creeping up against the close button. I thought there was only the close button in the header. Nevertheless, would it be better if I gave it another class, or added margin to the title itself?

Reply:
There is an optional back button as well.

Adding the margin to the title seems reasonable to me!

Reply:
Added `margin-right` to the title in b397aa5